### PR TITLE
Authentication screens added with formatted files and new base classes

### PR DESCRIPTION
**Pull request by dev-enforced**
In this Pull Request one can find the prettier formatted code files which consist of :

1) Screens for Login Page: https://deploy-preview-6--chillar-party.netlify.app/pages/auth-login.html
2) Screens for SignUp Page : https://deploy-preview-6--chillar-party.netlify.app/pages/auth-signup.html
3) Logout Button Usage in Wishlist and Cart management screens: 
https://deploy-preview-6--chillar-party.netlify.app/pages/cart-management.html
https://deploy-preview-6--chillar-party.netlify.app/pages/wishlist.html

Code also consists of classes related to flex-wrap,text-center and different buttons
Main files where major changes have been made in goes under the pages/auth-signup.html and pages/auth-login.html and the stylesheet in styles/authentication.css

### CODE REVIEW

File: pages/auth-login.html [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+            <div
+              class="submit-action gentle-flex-gap flex-column flex-justify-center"
+            >
+              <button
+                type="submit"
+                class="btn btn-info btn-info-hover submit-btn"
+              >
+                <a href="#" class="link-none"> LOG IN </a>
+              </button>
+              <button class="btn btn-link">
```
Comment (JantuDeb) [inline pages/auth-login.html]: Avoid adding `<a>` inside button, links may not clicked when you clicked on button.

**Review by MonikaJhingan — APPROVED** [review]
the code looks good to be merged ,color selection needs to be done carefully.

File: pages/auth-login.html [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+      <section class="main-content flex-column flex-center">
+        <div
+          class="main-content-container flex-column flex-justify-center p-10"
+        >
+          <div class="form-name text-center">
+            <h3>LOGIN</h3>
+          </div>
+          <form class="gentle-form-group gentle-flex-gap txt-sm">
+            <div class="gentle-input-group">
+              <label class="gentle-input-label">Email Address</label>
```
Comment (JantuDeb) [inline pages/auth-login.html]: Use `for` attribute when using a label for input.

File: pages/auth-signup.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+      <section class="main-content flex-column flex-center">
+        <div class="main-content-container flex-column flex-center p-10">
+          <div class="form-name">
+            <h3>SIGNUP</h3>
+          </div>
+          <form class="gentle-form-group gentle-flex-gap txt-sm">
+            <div class="gentle-flex-gap name-taker flex-wrap">
+              <div class="gentle-input-group">
+                <label class="gentle-input-label">First Name</label>
```
Comment (JantuDeb) [inline pages/auth-signup.html]: Same here use `for` attribute

File: pages/auth-signup.html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+                <input
+                  type="text"
+                  class="gentle-input"
+                  placeholder="e.g. Ashok"
+                />
+              </div>
+              <div class="gentle-input-group">
+                <label class="gentle-input-label txt-sm">Last Name</label>
```
Comment (JantuDeb) [inline pages/auth-signup.html]: Here also.

File: pages/auth-signup.html [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+              </label>
+            </div>
+            <div
+              class="submit-action gentle-flex-gap flex-column flex-justify-center"
+            >
+              <button
+                type="submit"
+                class="btn btn-info btn-info-hover submit-btn"
+              >
+                <a href="#" class="link-none"> CREATE NEW ACCOUNT </a>
```
Comment (JantuDeb) [inline pages/auth-signup.html]: Don't place `<a>` inside a button

**Review by JantuDeb — APPROVED** [review]
Overall the code is nice and clean. Also I added some suggestions than you can improve.

**Pull request merged**